### PR TITLE
Frecar/add licence

### PR DESCRIPTION
**Pull request by frecar**
@tiborsimko: Please review this.

### CODE REVIEW

**Comment by frecar** [comment]
Maybe it is better to include one LICENCE file and just refer to it from all the other files?

File: LICENCE-header.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+====
+    This file is part of obelix.
+
+    obelix is free software: you can redistribute it and/or modify
```
Comment (tiborsimko) [inline LICENCE-header.txt]: s/obelix/Obelix here and everywhere?

Comment (frecar) [inline LICENCE-header.txt]: Agreed

File: LICENCE-header.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
+====
+    This file is part of obelix.
```
Comment (tiborsimko) [inline LICENCE-header.txt]: We need to have a Copyright statement in every file, I can generate these using my scripts, as promised some time ago...

Comment (frecar) [inline LICENCE-header.txt]: The maven plugin will ensure that the file is included inn all scripts, so no worries.

**Comment by tiborsimko** [comment]
> Maybe it is better to include one LICENCE file and just refer to it from all the other files?

In GNU projects, it's usually both, i.e. (1) you have a LICENSE file (usually called COPYING) containing verbatim copy of the license text, and (2) you also include short statements in every file, because every file may have different copyright years etc.

**Comment by frecar** [comment]
Please review again @tiborsimko

**Comment by tiborsimko** [comment]
One more thing: the `COPYING` file should be the verbose copy of the full GNU GPLv3 license, see for example:

- https://github.com/inveniosoftware/claimstore/blob/master/LICENSE

Otherwise LGTM and :shipit: and I can add missing Copyright statements later.

**Pull request merged**